deepdrivemd/parsl.py

```diff
@@ -83,4 +83,49 @@ def config_factory(self, run_dir: PathLike) -> Config:
         )
 
 
-ComputeSettingsTypes = Union[LocalSettings, WorkstationSettings]
+class LSFStJudeSettings(BaseComputeSettings):
+    """Compute settings for LSF-based cluster at St. Jude."""
+
+    name: Literal["lsf"] = "lsf"  # type: ignore[assignment]
+    """Name of the platform."""
+    queue: str = "dgx"
+    """Name of the scheduler queue to submit to."""
+    available_accelerators: int = 8
+    """Number of GPU accelerators to use.
+    If running multi-node, set available_accelerators = 8
+    Else, number of simulations to run + 1"""
+    walltime: str = "00:10:00"
+    """Walltime."""
+    label: str = "htex"
+    """Label for the HighThroughputExecutor."""
+
+    def config_factory(self, run_dir: PathLike) -> Config:
+        num_gpus = self.available_accelerators
+        return Config(
+            run_dir=str(run_dir),
```

Comment:
I added the run_dir logic so the parsl logs should output to the experiment directory now.